source/_docs/maintain-custom-upstream.md

```diff
@@ -0,0 +1,263 @@
+
+## Create Test Site on Pantheon
+
+1. From your User Dashboard, click **Create New Site**.
+2. Name your site.
+3. Select your organization from the dropdown menu.
+4. Click **Create Site**.
+5. Select your Custom Upstream from the Organization Distribution framework options.
+6. Click **Visit your Pantheon Dashboard**.
+7. Click **Visit Development Site** and complete the installation process for the selected framework.
+
+## Test and Release Pantheon Core Updates
+
+1. Add Pantheon's Upstream as a [remote](https://git-scm.com/docs/git-remote) within a local clone of your Custom Upstream repository if you haven't done so already:
+
+    <!-- Nav tabs -->
+    <ul class="nav nav-tabs" role="tablist">
+      <li id="wptab1" role="presentation" class="active"><a href="#wp1" aria-controls="wp1" role="tab" data-toggle="tab">WordPress</a></li>
+      <li id="d8tab1" role="presentation"><a href="#d81" aria-controls="d81" role="tab" data-toggle="tab">Drupal 8</a></li>
+      <li id="d7tab1" role="presentation"><a href="#d71" aria-controls="d71" role="tab" data-toggle="tab">Drupal 7</a></li>
+    </ul>
+
+    <!-- Tab panes -->
+    <div class="tab-content">
+    <div role="tabpanel" class="tab-pane active" id="wp1">
+    <pre id="git-pull-wp"><code class="command hljs" data-lang="">git remote add pantheon-wordpress git://github.com/pantheon-systems/WordPress.git</code></pre>
+    </div>
+    <div role="tabpanel" class="tab-pane" id="d81">
+    <pre id="git-pull-drops-8"><code class="command hljs" data-lang="">git remote add pantheon-drops-8 git://github.com/pantheon-systems/drops-8.git</code></pre>
+    </div>
+    <div role="tabpanel" class="tab-pane" id="d71">
+    <pre id="git-pull-drops-7"><code class="command hljs" data-lang="">git remote add pantheon-drops-7 git://github.com/pantheon-systems/drops-7.git</code></pre>
+    </div>
+    </div><br>
+
+2. We'll need to do the same for your new test site on Pantheon. Let's grab the site's repository URL on Pantheon using [Terminus](/docs/terminus):
+
+    ```command
+    terminus connection:info <site>.<dev> --field=git_url
+    ```
+
+
+  Replace `<git_url>` in the following command to add your new test site as a [remote](https://git-scm.com/docs/git-remote):
+
+    ```command
+    git remote add pantheon-test <git_url>
+    ```
+
+
+3. Checkout a new branch:
+
+    ```command
+    git checkout -b core-update
+    ```
+
+    It's important to use feature branches when applying and testing updates. Updates applied on the master branch and pushed to the remote repository on GitHub or Bitbucket become available to individual sites downstream. Using a feature branch gives us a chance to reveal issues before distributing updates.
+
+4. Pull down Pantheon's core updates from the appropriate upstream:
```

Comment:
The commands in the tabs below rebase against master. Based on the preceding paragraph, wouldn't we want to rebase `pantheon-<CMS>` to the `core-update` branch?